caikit/core/model_management/model_trainer_base.py

```diff
@@ -44,6 +44,9 @@ def __init__(self, *args, **kwargs):
             kwargs["future_name"] = kwargs["trainer_name"]
         if "training_id" in kwargs:
             kwargs["future_id"] = kwargs["training_id"]
+        # If save path is provided then add it as a property
```

Comment:
I think we might want to do this no matter what and just assign it to `None` given that the log line will fire on _any_ trainer?

Reply:
That's a good point. The save_path should almost always be included in future construction but that does rely on the trainer implementations